### PR TITLE
Updating bundle dependencies

### PR DESCRIPTION
**Pull request by StorytellerCZ**
Related to my Typescript updates I would like to consider updating dependencies in the bundles. Here are some which I think are super easy to update. Starting with a patch version of NPM.

### CODE REVIEW

File: npm-packages/eslint-plugin-meteor/scripts/build-dev-bundle-common.sh
Comment (StorytellerCZ) [inline npm-packages/eslint-plugin-meteor/scripts/build-dev-bundle-common.sh:1]: Found this file. Seems like copy of `build-dev-bundle-common.sh` from scripts. Is this really needed?

Comment (StorytellerCZ) [reply npm-packages/eslint-plugin-meteor/scripts/build-dev-bundle-common.sh:1]: Same for `dev-bundle-tool-package.js`

Comment (Grubba27) [reply npm-packages/eslint-plugin-meteor/scripts/build-dev-bundle-common.sh:1]: those files inside of eslint I think is not needed updating, also I think it should be deleted but I forgot which PR was removing them

Comment (StorytellerCZ) [reply npm-packages/eslint-plugin-meteor/scripts/build-dev-bundle-common.sh:1]: For now I'll keep it. If there is a PR that removes them, then only better.

**Pull request closed without merging**